charts/cluster-api-runtime-extensions-nutanix/README.md

```diff
@@ -32,9 +32,9 @@ A Helm chart for cluster-api-runtime-extensions-nutanix
 | deployment.replicas | int | `1` |  |
 | env | object | `{}` |  |
 | helmAddonsConfigMap | string | `"default-helm-addons-config"` |  |
-| helmRepositoryImage.pullPolicy | string | `"IfNotPresent"` |  |
-| helmRepositoryImage.repository | string | `"ghcr.io/nutanix-cloud-native/caren-helm-reg"` |  |
-| helmRepositoryImage.tag | string | `""` |  |
+| helmChartsImage.pullPolicy | string | `"IfNotPresent"` |  |
+| helmChartsImage.repository | string | `"ghcr.io/nutanix-cloud-native/caren-helm-reg"` |  |
+| helmChartsImage.tag | string | `""` |  |
```

Comment:
Should we keep the current values to avoid a breaking change, the image name is not changing and is now doing both the serving and in the init container?

Reply:
Since this is deployed using clusterctl and the runtime-extensions config file, we don't have to worry about this.

Reply:
Thats true for Konvoy but this is an open source project and there are already [docs](https://confluence.eng.nutanix.com:8443/display/KAR/Create+CAPX+Cluster+using+CAREN+and+Install+Kommander) that have instructions to use it as a Helm chart.
I understand the reasoning why you changed it in the first place when the image only contained the Helm charts, but now thats also has mindthegap and is used as a repository going back to `helmRepositoryImage` is a little more accurate IMO, but has the benefit of not changing the public interface, when not needed.

Reply:
deploying via helm chart is for development purposes and the value already has sane defaults. if someone was deploying this in their own developer workflow they are well equipped to read the docs and change the values as needed.